shared/team-building/container.tsx

```diff
@@ -155,8 +154,9 @@ const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
   const userResults: Array<Types.User> =
     teamBuildingState.searchResults.get(trim(ownProps.searchString))?.get(ownProps.selectedService) ?? []
 
-  const preExistingTeamMembers: I.Map<string, MemberInfo> =
-    (ownProps.teamname && state.teams.teamNameToMembers.get(ownProps.teamname)) || I.Map<string, MemberInfo>()
+  const preExistingTeamMembers: Map<string, MemberInfo> =
+    (ownProps.teamname && state.teams.teamNameToMembers.get(ownProps.teamname)) ||
```

Comment:
`??`

Reply:
That'd allow `preExistingTeamMembers === ''` (which TS caught, that's nice)